features/networking/ovn.feature

```diff
@@ -0,0 +1,216 @@
+Feature: OVN related networking scenarios
+
+  # @author rbrattai@redhat.com
+  # @case_id OCP-26139
+  @admin
+  @destructive
+  Scenario: Repeat delete and creation
```

Comment:
@rbbratta The title should match the title for OCP-26139 in Polarion. Either modify this to match Polarion or vice versa

Reply:
added new test OCP-28936 Create/delete pods while forcing OVN leader election and updated comment